scripts/install/install-pocs.sh

```diff
@@ -132,7 +143,7 @@ function setup_env_vars {
     ENV_FILE="${PANDIR}/env"
```

Comment:
Let's move this line to the top of the file with the other variables.  See also the note below.